src/routes/api/installation/api-installation-delete.ts

```diff
@@ -4,6 +4,7 @@ import { getJiraClient } from "~/src/jira/client/jira-client";
 
 export const ApiInstallationDelete = async (req: Request, res: Response): Promise<void> => {
 	const githubInstallationId = req.params.installationId;
```

Comment:
Nit: all casing for GH should be gitHub.

Reply:
fixed.